Converting community-wide-resources to rst

Iqmaa opened the pull request:
fixed #221 
I have been able to convert 'community wide resources' to rst format, excluding the page's metadata

RCheesley commented on docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.rst:
This is a revision file (indicated by .revs) so it should be ignored.

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,282 @@
+
+Your agenda might now look something like this:
+
+::
```
This markup looks a little wonky.

RCheesley reviewed (changes requested):
Hey there! Thanks for making a great start on this!

A few things that you need to factor into this PR before we can merge - these are all documented in the issue - please take the time to read it:

1. Please follow our headings syntax, as outlined in the issue description. We don't use over and under headings, only under.
2. Please don't touch the .revs folder - if anything they should be deleted. It's the old version control system Grav used, and not the live files.
3. You've generated link files but you're not using them properly, please check the docs here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-the-external-links-extension
4. We don't need to use random lines like you've done in the async meetings file on line 5 - they don't serve a purpose, so you can remove them.
5. Please check that all links are using the link macro, e.g. on line 59 of the same file there's a link posted.
6. Please use ref: or doc: to link to other pages.
7. Please use the image markup here: https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html#add-images rather than figure.

Let me know if you are stuck with anything!

Iqmaa commented:
Hey @RCheesley, sorry for that.
I have:

1. Fixed the headings issue
2. Converted the .rev files back to original form
3. Corrected all linking implementation issues
4. removed unnecessary lines
5. and replaced the figure with the image markup

Please let me know if there are any other changes I need to make.

Iqmaa commented:
Hey @RCheesley, sorry for that.
I have:

1. Fixed the headings issue
2. Converted the .rev files back to original form
3. removed unnecessary lines
4. and replaced the figure with the image markup

But I am having an issue with the external linking. Although I have implemented the changes according to the instructions the links aren't showing as they should be, do you perhaps know why this is?

RCheesley commented on docs/pages/07.community-wide-resources/.revs/20200310-170715/chapter.md:
@Iqmaa these don't need to be pages/* any more, in fact in the index.rst we've declated that they're in resources/* so I will move these for you and make a commit directly, to save us having to go through those changes. Hold on to your hat :)

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+How to prepare & run a Mautic Team Meeting
```
```suggestion
How to prepare & run a Mautic team meeting
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+How to prepare & run a Mautic Team Meeting
+==========================================
+
+Meeting cycle; Async vs. Live meetings
```
```suggestion
Meeting cycle; async vs. live meetings
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+Preparing for a meeting
+=======================
+
+High-Level Agenda
```
```suggestion
High-level agenda
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+Don’t forget: Please do experiment! Let the structure improve over time,
+and of course: Adopt to temporary circumstances.
+
+Specific Topics within the Agenda
```
```suggestion
Specific topics within the agenda
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+2. Review Trello board 2.1 Ensure all teams have an onboarding
+   person/workflow …
+
+Make Agenda items stick out in Slack
```
```suggestion
Make agenda items stick out in Slack
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+   :two:  Review Trello board
+   :two:.:one: Ensure all teams have an onboarding person/workflow
+   ...
+Post meeting invitation and Agenda in Slack
```
```suggestion

Post meeting invitation and Agenda in Slack
```

Usually code blocks require a clean line break after them.

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+   For continuation (and also for everybody who can not make it to the live call) we will turn every meeting topic into a thread right here in Slack, so you can still join the asynchronous follow-up discussion whenever you are available and as often as you can, within the next 24 hours.
+   /cc @teammember1 @teammember2 @teammember3 ...
+
+Prepare your Meeting content
```
```suggestion
Prepare your meeting content
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/12.how-to-prepare-and-run-a-mautic-team-meeting/docs.rst:
```diff
@@ -0,0 +1,273 @@
+
+Use the :doc:`meeting parser tool </community-wide-resources/meetings/asynchronous-async-meetings#exporting-slack-meetings>`
+
+Live Meetings
```
```suggestion
Live meetings
```

RCheesley commented on docs/pages/07.community-wide-resources/02.meetings/chapter.rst:
```diff
@@ -0,0 +1,8 @@
+# Meetings
```
```suggestion
Meetings
```

Might ned to correct the underline heading once tweaking this.

RCheesley commented on docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:
```diff
@@ -0,0 +1,279 @@
+serious burnout takes its grip, it can destroy families, careers, and
+many other aspects of life.
+
+Detecting and Treating Burnout
```
```suggestion
Detecting and treating burnout
```

RCheesley commented on docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:
```diff
@@ -0,0 +1,279 @@
+required, but it is often emotionally required inside the mind of the
+contributor.
+
+Work/Life Balance
```
```suggestion
Work/life balance
```

RCheesley commented on docs/pages/07.community-wide-resources/03.avoiding-burnout/docs.rst:
```diff
@@ -0,0 +1,279 @@
+can’t or doesn’t need to, but stand firm: it is for his own good, and he
+will thank you for it.
+
+VOLUNTEERISM ESCAPES NOTHING:
```
```suggestion
Volunteerism escapes nothing:
```

RCheesley commented on docs/pages/07.community-wide-resources/04.getting-help-in-a-crisis/docs.rst:
```diff
@@ -0,0 +1,44 @@
+and that you are concerned. Then encourage them to contact one of the
+resources listed here to get the professional help that they need.
+
+Prevention Hotlines
```
```suggestion
Prevention hotlines
```

RCheesley commented on docs/pages/07.community-wide-resources/chapter.rst:
This one isn't required, we don't need a chapter any more.

RCheesley reviewed (changes requested):
A few changes but mostly the same things:

- Not using camel case in headings
- A file not required
- A missed link macro

I will make a commit to move all the files to the right folder so you don't need to worry about that, and will also write some half-decent onboarding docs on how to write for the docs as I appreciate this is somewhat lacking!

RCheesley commented:
LMK when you're done @Iqmaa if you'd like me to fix the RST issues/move the directories!

RCheesley commented:
OK Vale is now happy (I have checked it locally, as the linter is having a problem which I'll try to fix in another PR) - the build is now passing. LGTM! Thanks so much for getting this underway @Iqmaa and for your patience! 🚀

Iqmaa commented:
I'm really grateful for your patience and help @RCheesley, I learnt quite a lot from contributing to this.
Thanks again !
